packages/core/src/api/blockManipulation/commands/moveBlocks/moveBlocks.ts

```diff
@@ -154,22 +169,31 @@ function flattenColumns(
  * reference block.
  */
 export function moveSelectedBlocksAndSelection(
-  editor: BlockNoteEditor<any, any, any>,
+  tr: Transaction,
```

Comment:
looks like we could go either way here, right?
- operate on a `transaction`, like you did, and the lower-level (internal) BlockNote methods
- vs. `editor.transact` and use higher level BlockNote methods. If I see it correctly, this is also valid option because we don't use lower-level prosemirror operations for `moveBlocks` related methods

Reply:
Yep, both are valid. I think the code is clearer as the high-level methods. So, I'll put it back to those